packages/web/package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "scripts": {
     "dev": "concurrently \"next\" \"yarn generate --watch\"",
+    "serve": "concurrently \"next start\" \"watch --interval 300 'yarn build'\"",
```

Comment:
why this change?

Reply:
The `dev` command was running out of memory on the smallest render.com instance, so I was trying to mimic the behavior of production which is static but intermittently regenerated.

I can pull it out, it isn't being used by anyone including myself.

Reply:
Done.